packages/nodes-base/nodes/Xero/Xero.node.ts

```diff
@@ -285,7 +285,7 @@ export class Xero implements INodeType {
 						const lineItems: ILineItem[] = [];
 						for (const lineItemValue of lineItemsArray) {
 							const lineItem: ILineItem = {
-								Tracking: [],
+								Tracking: lineItemValue.tracking as [],
```

Comment:
That doesn't seem like the right type. What is `tracking`? I'm thinking you either want `as string[]` or `[lineItemValue.tracking]`. `[]` is an empty tuple that doesn't need a type.

Reply:
@Ken-Michalak according to the Xero node's Interface file Tracking is marked as a `Tracking?: IDataObject[];`

Reply:
@Ken-Michalak updated